_posts/2022-08-11-temp-title/my-4-biggest-internship-takeaways.Rmd

```diff
@@ -0,0 +1,50 @@
+
+```{r}
+tweetrmd::include_tweet("https://twitter.com/hbomaxhelp/status/1405712235108917249?lang=en")
+```
+
+After reading that, I made sure to double and triple check everything I did so that my work would not result in a catastrophe such as that. Through the guidance of my co-workers, I was able to not only fix my mistakes, but also learn from them and use that knowledge for future assignments. Everyone makes mistakes and that is okay, but what is not okay is not learning from those mistakes and continuing to make them. Getting caught up and upset about a mistake is not going to make you a better person/worker if you don't do anything about it afterwards. The purpose of an internship is to grow and learn so that you can make fewer mistakes when you have a full-time job. So really, mistakes are crucial to the learning process and are the reason successful people are successful.\
+
+### 3. How to work in a professional environment
+
+What I have learned about working in a professional environment is that being a working professional does not mean you have to be professional all the time. What really put me at ease were the amount of "lol"s and "haha"s I received in Teams messages and emails. It reminded me that people aren't just their jobs and that they have personalities. This could just be the PCCTC, but I never felt pressured to dress or write or act a certain way, which was really nice to see because using slang has largely been considered unprofessional. I feel like as the younger generations start getting into the professional world, rules that used to be strongly upheld will start to relax because those don't really matter. For example, in my opinion, dressing up very nicely for meetings should not be so strongly supported as long as you are getting your work done. If I get 500 lines of code done in 4 hours wearing sweatpants, I'm still going to get those 500 lines of code done in 4 hours wearing a pantsuit - I'll just be more uncomfortable. But again, these are my opinions and people can freely disagree. <br> <br> Back to the topic at hand: working in a professional environment. For the rest of your life, most of what you get and don't get is a direct result of connections so establishing these connections whenever you can is super important when you're working with professionals. You also learn how to speak with others in a professional setting which is key for job interviews. In sum, working in a professional environment is a lot less scary than I was anticipating once I realized that working professionals still have personalities and aren't these corporate robots that criticize my every move.
```

Comment:
`This could just be the PCCTC, but I never felt pressured to dress or write or act a certain way, which was really nice to see because using slang has largely been considered unprofessional.`

This sentence seems to be two separate thoughts? And maybe it is more that you thought slang would be unprofessional, not the claim or largely been considered unprofessional?

Reply:
`You also learn how to speak with others in a professional setting which is key for job interviews.`

This also seems a bit out of place, because I think the level of professionalism required for a job interview is a bit different than your day to day interactions once you have the job.